csa-runner-app/src/main/java/com/farao_community/farao/swe_csa/app/dichotomy/DichotomyRunner.java

```diff
@@ -177,6 +175,17 @@ public RaoResult runDichotomy(CsaRequest csaRequest) throws GlskLimitationExcept
         }
     }
 
+    double getMaxCounterTrading(CounterTradeRangeAction ctra1, CounterTradeRangeAction ctra2, double initialExchange1, double initialExchange2, String borderName) {
```

Comment:
I suggest to use only ctRaTowardsEs  and initialExchangeTowardsES and borderName, as parameters and invert the sign inside the method

Reply:
I change to use only initialExchange towards ES, but I think I can't do the same thing for ctRa, which are limits defined by TSOs and could be different for the 2 countries, even if it's the same border, they could have different counstraits ?